src/trade/types.ts

```diff
@@ -0,0 +1,80 @@
+import { Address } from 'viem';
+
+/**
+ * Note: exported as public Type
+ */
+export type Token = {
+  address: Address;
+  decimals?: number;
+  name?: string;
+  logoURI?: string;
+  currencyCode?: string;
+};
+
+/**
+ * Note: exported as public Type
+ */
+export type TrendingToken = Token & {
+  numOfBuys: number;
+  numOfSells: number;
+};
+
+/**
+ * Note: exported as public Type
+ */
+export type QuoteWarning = {
+  Type?: string;
+  Message?: string;
+  Description?: string;
+};
+
+/**
+ * Note: exported as public Type
+ */
+export type Quote = {
+  fromAsset: Token;
+  toAsset: Token;
+  fromAmount: string;
+  toAmount: string;
+  amountReference: string;
+  priceImpact: string;
+  highPriceImpact: boolean;
+  slippage: string;
+  warning?: QuoteWarning;
+};
+
+/**
+ * Note: exported as public Type
+ */
+export type Trade = {
+  tx: Transaction;
+  fee: Fee;
+  approveTx?: Transaction;
+  chainId: string;
+};
+
+/**
+ * Note: exported as public Type
```

Comment:
Let's add the comments only when we actually exported, right now you are not exporting them.